packages/rich-text-editor/src/vaadin-rich-text-editor.js

```diff
@@ -1122,6 +1122,13 @@ class RichTextEditor extends ElementMixin(ThemableMixin(PolymerElement)) {
     }
   }
 
+  /** @protected */
+  disconnectedCallback() {
```

Comment:
Let's place this callback closer to the top as we generally do it, e.g. next to `attributeChangedCallback`.